browsepy/compat.py

```diff
@@ -248,7 +249,16 @@ def pathconf(path,
     '''
 
     if pathconf_fnc and pathconf_names:
-        return {key: pathconf_fnc(path, key) for key in pathconf_names}
+        pathconf_output = {}
+        for key in pathconf_names:
+            try:
+                pathconf_output[key] = pathconf_fnc(path, key)
+            except OSError as exc:
+                if exc.errno == errno.EINVAL:
+                    pass  # Ignoring unsupported pathconf key
+                else:
+                    raise exc from None
```

Comment:
This whole if-pass-else-rraise construct is unnecessary, please just do if-raise as in
```python
if exc.errno != errno.EINVAL:
    raise
```